### PR TITLE
[SDK]increase timeout for automatic runtime and emphasize user can continue with retry

### PR DESCRIPTION
**Pull request by D-W-**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

image

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by wangchao1230 — DISMISSED** [review]

**Review by lalala123123 — APPROVED** [review]

File: src/promptflow/promptflow/azure/_constants/_flow.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class FlowJobType:
 
 CLOUD_RUNS_PAGE_SIZE = 25  # align with UX
 
-SESSION_CREATION_TIMEOUT_SECONDS = 10 * 60  # 10 minutes
+SESSION_CREATION_TIMEOUT_SECONDS = 20 * 60  # 20 minutes
```
Comment (wangchao1230) [inline src/promptflow/promptflow/azure/_constants/_flow.py:23]: Lisa comment that we should not change this, as we control the user experience, and should not change this as this not improves the user experience.  @D-W-  @eniac871 

However we should make this configurable for Runner,

Comment (D-W-) [reply src/promptflow/promptflow/azure/_constants/_flow.py:23]: Added env var SESSION_CREATION_TIMEOUT_SECONDS to allow runner configure

**Review by brynn-code — DISMISSED** [review]

File: src/promptflow/promptflow/azure/_constants/_flow.py [hunk 2]
```diff
@@ -21,6 +21,7 @@ class FlowJobType:
 CLOUD_RUNS_PAGE_SIZE = 25  # align with UX
 
 SESSION_CREATION_TIMEOUT_SECONDS = 10 * 60  # 10 minutes
+SESSION_CREATION_TIMEOUT_ENV_VAR = "SESSION_CREATION_TIMEOUT_SECONDS"
```
Comment (wangchao1230) [inline src/promptflow/promptflow/azure/_constants/_flow.py:24]: add a PROMPTFLOW_ prefix?

File: src/promptflow/promptflow/azure/_restclient/flow_service_caller.py [hunk 3]
```diff
@@ -500,11 +500,23 @@ def create_flow_session(
...
+                        SESSION_CREATION_TIMEOUT_ENV_VAR, os.environ.get(SESSION_CREATION_TIMEOUT_ENV_VAR)
+                    )
+                )
         # InProgress is only known non-terminal status for now.
         while status in [None, "InProgress"]:
             if time_run + sleep_period > timeout_seconds:
-                message = f"Timeout for session {action} {session_id} for {AUTOMATIC_RUNTIME}.\n" \
-                          "Please resubmit the flow later."
+                message = f"Polling timeout for session {session_id} {action} for {AUTOMATIC_RUNTIME}.\n" \
+                          f"To proceed the {action} for {AUTOMATIC_RUNTIME}, you can retry using the same flow, " \
```
Comment (0mza987) [inline src/promptflow/promptflow/azure/_restclient/flow_service_caller.py:518]: Shall we inform how long the timeout peried was? Like the value of above `timeout_seconds`

Comment (crazygao) [reply src/promptflow/promptflow/azure/_restclient/flow_service_caller.py:518]: by the way, should we sync this value with real session timeout?

**Review by brynn-code — APPROVED** [review]

**Review by zhengfeiwang — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**